yarn.lock

```diff
@@ -18666,6 +18666,11 @@ google-gax@~1.12.0:
     semver "^6.0.0"
     walkdir "^0.4.0"
 
+google-libphonenumber@^3.2.15:
+  version "3.2.15"
+  resolved "https://registry.yarnpkg.com/google-libphonenumber/-/google-libphonenumber-3.2.15.tgz#3a01dc554dbf83c754f249c16df3605e5d154bb9"
+  integrity sha512-tbCIuzMoH34RdrbFRw5kijAZn/p6JMQvsgtr1glg2ugbwqrMPlOL8pHNK8cyGo9B6SXpcMm4hdyDqwomR+HPRg==
+
 google-libphonenumber@^3.2.4:
```

Comment:
We should only have one instance of `google-libphonenumber`.

`yarn` has trouble keeping only one instance of compatible versions, when this happens you can manually delete both and re-run `yarn`.